addons/website/views/snippets/snippets.xml

```diff
@@ -414,7 +414,7 @@
          data-selector="#wrapwrap > header"
          data-no-check="true"
          groups="website.group_website_designer">
-        <we-checkbox string="Show Sign In" data-customize-website-views="portal.portal_show_sign_in" data-reload="/"/>
+        <we-checkbox string="Sign In" data-customize-website-views="portal.portal_show_sign_in" data-reload="/"/>
```

Comment:
Like I indicated in the commit message, reordering of the options will be done later otherwise I will have too many conflicts (so I only kept the relabeling). However, when you do that, please always do the move in a dedicated commit. Otherwise it is impossible to review, I lost a lot of time spliting your work to understand it :/ The first commit of your PR was a 8-10 commits work.